app/scripts/components/content/content.jsx

```diff
@@ -23,10 +25,13 @@ var Content = React.createClass({
   render: function() {
     var content, comments, title;
 
-    if (_.isUndefined(this.props.data)) {
-      return (<i className="fa fa-refresh fa-spin fa-5x"></i>);
+    if (_.isNull(this.props.data)) {
```

Comment:
Not to distract you, but we could probably write the spinner as a mixin that does this.